### PR TITLE
Noel c9 patch 5

### PR DESCRIPTION
**Pull request by NoelC9**
This fixes bulk orders and improves the functionality of our bar graph class. You can now chose a date and have the bar graph repaint itself dynamically

### CODE REVIEW

File: saleorders.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
-10232020, abc@gmail.com, arvada, apple,10
-12122020, bcd@msudenver.edu, littleton, banana,20
-04292020, efg@gmail.com, fort collins, pear,50
-02232019, lame@game.com, Denver, grapes,5
-05242002, same@name.com, nepal, guava,15
\ No newline at end of file
+10/23/2020, abc@gmail.com, arvada, apple,10
```
Comment (jgonzo3224) [inline saleorders.txt:1]: Nice change to the date format.

File: src/main/java/com/cs3250p1/project1/Main.java [hunk 2]
```diff
@@ -245,49 +254,63 @@ public void actionPerformed(ActionEvent e) {
             List<String> lines = gui.getStringFromFileDialog(); //lets user choose a file,
             // then loads it's string as a list onto the variable
             //parse that string and store it into a structure
-
```
Comment (jgonzo3224) [inline src/main/java/com/cs3250p1/project1/Main.java:248]: Nice adding in the array and time and date fixes.

File: src/main/java/com/cs3250p1/project1/SalesOrder.java [hunk 3]
```diff
@@ -8,7 +10,7 @@
     protected int quantity;*/
 
     protected String supplier_id;
-    protected String date; //dates are long so needs double
+    protected Timestamp date; //dates are long so needs double
     protected String email;
```
Comment (jgonzo3224) [inline src/main/java/com/cs3250p1/project1/SalesOrder.java:14]: Code changes from a string to using the timestamp import function.

**Review by jgonzo3224 — APPROVED** [review]
All these changes look good. The timestamp looks good for the final project.

File: src/main/java/com/cs3250p1/project1/BarChartEx.java [hunk 4]
```diff
@@ -1,77 +1,95 @@
 package com.cs3250p1.project1;
```
Comment (dgiri12) [inline src/main/java/com/cs3250p1/project1/BarChartEx.java:1]: Alright, nice to get the final version of this bar graph, looks great and fulfills the user story

File: src/main/java/com/cs3250p1/project1/Main.java [hunk 5]
```diff
@@ -4,9 +4,16 @@
```
Comment (dgiri12) [inline src/main/java/com/cs3250p1/project1/Main.java:4]: Improved speed of bulk order processing is nice. Communication with sql table is good. Supplier order is good as it is, I think that won't need changing and we can just go wth it

**Review by dgiri12 — APPROVED** [review]
Overall changes are good, we can carry onto the next step, i.e. adding in the new reformatted gui code

**Pull request merged**